map.go

```diff
@@ -379,6 +385,31 @@ func NewMapFromExistingMapById(id int) (*EbpfMap, error) {
 	return NewMapFromExistingMapByFd(int(fd))
 }
 
+// NewMapFromExistingMapMapByPath creates eBPF map from a pinned BPF object path.
+// Pinned BPF object is a kernel mechanism to let non owner process to use BPF objects.
+// Common use case - tooling for troubleshoot / inspect existing BPF objects in the kernel.
+func NewMapFromExistingMapByPath(path string) (*EbpfMap, error) {
```

Comment:
Could you please add integration test for `NewMapFromExistingMapByPath` here:
https://github.com/dropbox/goebpf/blob/master/itest/map_test.go

We don't run integration tests on Travis CI (it does not allow using eBPF), you can run it manually on your setup by:
```bash
$ cd itest
$ sudo make test
././itest_test -test.v
....
=== RUN   TestMapSuite
=== RUN   TestMapSuite/TestArrayOfMaps
=== RUN   TestMapSuite/TestHashOfMaps
=== RUN   TestMapSuite/TestMapArrayInt
=== RUN   TestMapSuite/TestMapArrayInt16
=== RUN   TestMapSuite/TestMapArrayUInt64
=== RUN   TestMapSuite/TestMapDoubleClose
=== RUN   TestMapSuite/TestMapFromExistingByFd
=== RUN   TestMapSuite/TestMapHash
=== RUN   TestMapSuite/TestMapLPMTrieIPv4
=== RUN   TestMapSuite/TestMapLPMTrieIPv6
=== RUN   TestMapSuite/TestMapPersistent
=== RUN   TestMapSuite/TestMapProgArray
--- PASS: TestMapSuite (0.19s)
....
```